test/libyul/yulSyntaxTests/objects/datacopy.yul

```diff
@@ -0,0 +1,8 @@
+{
+    datacopy(0, 1, 2)
+
+    // datacopy also accepts pretty much anything which can be turned into a number
```

Comment:
A bit of a weird comment :-). It just accepts arbitrary yul expressions as arguments... but fine!